pages/dkp/kaptain/2.0.0/version-policy/index.md

```diff
@@ -34,22 +29,21 @@ Advice on preventing and recovering from failures and troubleshooting, as availa
 Support for Experimental components is provided on a Standard level, Severity 4 basis only.
 
 ## Istio Support Status
+
 Istio is a required prerequisite for Kaptain.
 Istio usage is only supported in conjunction with Kaptain and the configurations delivered as part of Kaptain.
 All other usage of [experimental]Istio[/experimental] is considered experimental.
 
-## Support for KUDO - Cassandra, Kafka & Spark
-### Support Definition - Secondary Support
-Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark, and additionally support for the KUDO-based operator of the mentioned platform services.
+## Secondary Support for Cassandra, Kafka & Spark
 
-*Base Technology refers to Cassandra, Kafka, and Spark.
+<p class="message--note"><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>
```

Comment:
is this the correct html for this? I think it should be like "info" not red like this seems to be?

Reply:
That would be:
```suggestion
<p class="message--note"><strong>NOTE: </strong><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>
```

Reply:
Looks good to me in front-end. What do you mean with red?

Reply:
nothing.. the message is purple, not red 🙈